release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,96 @@
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="1a,1a,1a",frame=none,grid=none]
+|===
+|A::
+xref:../networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc#aws-load-balancer-operator-release-notes[AWS Load Balancer (ALB) Operator]
+
+|C::
+xref:../security/cert_manager_operator/cert-manager-operator-release-notes.adoc#cert-manager-operator-release-notes[{cert-manager-operator}] +
+{nbsp} +
+xref:../observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc#cluster-observability-operator-release-notes[{coo-first}] +
+{nbsp} +
+xref:../security/compliance_operator/compliance-operator-release-notes.adoc#compliance-operator-release-notes[Compliance Operator] +
+{nbsp} +
+xref:../nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.adoc#nodes-cma-autoscaling-custom-rn[Custom Metrics Autoscaler Operator]
+
+|D::
+link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/release_notes/index[{rh-dev-hub} Operator]
+
+|E::
+xref:../networking/external_dns_operator/external-dns-operator-release-notes.adoc#external-dns-operator-release-notes[External DNS Operator]
+
+|F::
+xref:../security/file_integrity_operator/file-integrity-operator-release-notes.adoc#file-integrity-operator-release-notes-v0[File Integrity Operator]
+
+|K::
+xref:../nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc#nodes-descheduler-release-notes[{descheduler-operator}]
+
+|L::
+xref:../observability/logging/logging_release_notes/logging-5-9-release-notes.adoc#logging-5-9-release-notes[{logging-uc}]
+
+|M::
+xref:../migration_toolkit_for_containers/release_notes/mtc-release-notes.adoc#mtc-release-notes[{mtc-full} ({mtc-short})]
+
+|N::
+xref:../observability/network_observability/network-observability-operator-release-notes.adoc#network-observability-operator-release-notes[Network Observability Operator] +
+{nbsp} +
+xref:../security/nbde_tang_server_operator/nbde-tang-server-operator-release-notes.adoc#nbde-tang-server-operator-release-notes[Network-bound Disk Encryption (NBDE) Tang Server Operator]
+
+|O::
+xref:../backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc#oadp-1-4-release-notes[{oadp-first}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_dev_spaces/3.16#Release%20Notes[{openshift-dev-spaces-productname}] +
+{nbsp} +
+xref:../observability/distr_tracing/distr-tracing-rn.adoc#distr-tracing-rn[{DTProductName}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/1.13/html/release_notes/index[{gitops-title}] +
```

Comment:
```suggestion
link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/[{gitops-title}] +
```
The latest is https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/1.14, but the one I have suggested is neutral and not version-specific.